Split post-upgrade leapp steps into two (#2002)
theforeman/foreman-documentation: guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

----
endif::[]
ifdef::satellite[]
. Make sure that you follow the below steps in the _Upgrading from RHEL 7 to RHEL 8_ guide:  
.. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system]

> .. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/performing-post-upgrade-tasks-rhel-7-to-rhel-8_upgrading-from-rhel-7-to-rhel-8[Performing post-upgrade tasks]

Author: Hey @adamlazik1 so @evgeni and I had come to the conclusion that this should be a single sentence in https://github.com/theforeman/foreman-documentation/pull/1963.

> . Make sure that you follow the below steps in the _Upgrading from RHEL 7 to RHEL 8_ guide:  
> .. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system]
> .. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/performing-post-upgrade-tasks-rhel-7-to-rhel-8_upgrading-from-rhel-7-to-rhel-8[Performing post-upgrade tasks]

Reviewer: Oh I see. I do agree name of the guide probably does not need to be mentioned twice. On the other hand, we try to avoid sub steps if possible. Maybe we could use bullet points instead?

```suggestion
. Make sure that you complete these procedures in  _Upgrading from RHEL 7 to RHEL 8_ :  
* https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system]
* https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/performing-post-upgrade-tasks-rhel-7-to-rhel-8_upgrading-from-rhel-7-to-rhel-8[Performing post-upgrade tasks]
```

> .. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/performing-post-upgrade-tasks-rhel-7-to-rhel-8_upgrading-from-rhel-7-to-rhel-8[Performing post-upgrade tasks]

Reviewer 2: Well, I wasn't that much on the "it has to be one sentence" side, more "if we have two, we duplicate the very long name of the guide which feels odd".

The current change works for me, and the one Adam is proposing does too (in #1963 it was actually one super long sentence, see https://github.com/theforeman/foreman-documentation/pull/1963#pullrequestreview-1288809308), so :shrug:

> .. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/performing-post-upgrade-tasks-rhel-7-to-rhel-8_upgrading-from-rhel-7-to-rhel-8[Performing post-upgrade tasks]

Author: @evgeni agreed! It was a terribly long sentence.
@adamlazik1 the problem with using bullet points is that it might not convey that the steps need to be done in that particular order (first verify then post-upgrade tasks). What do you think?

> . Make sure that you follow the below steps in the _Upgrading from RHEL 7 to RHEL 8_ guide:  
> .. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system]
> .. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/performing-post-upgrade-tasks-rhel-7-to-rhel-8_upgrading-from-rhel-7-to-rhel-8[Performing post-upgrade tasks]

Reviewer: That is a valid point. So it seems sub steps cannot be avoided in this case. Then my final suggestion is just to simplify the introductory sentence, but other than that I suppose this is good to go.

```suggestion
. Complete these procedures in  _Upgrading from RHEL 7 to RHEL 8_:  
.. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system]
.. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/performing-post-upgrade-tasks-rhel-7-to-rhel-8_upgrading-from-rhel-7-to-rhel-8[Performing post-upgrade tasks]
```

> .. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/performing-post-upgrade-tasks-rhel-7-to-rhel-8_upgrading-from-rhel-7-to-rhel-8[Performing post-upgrade tasks]

Reviewer 2: LGTM!
